### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by mariarivera30**
- Update order of ide and jup[yter
- add rabbitmq picture
-add note about resource files

### CODE REVIEW

**Comment by RichardAF8** [comment]
First step should be forking the repo with an image of clicking the fork button

File: Tech-Lab-On-Campus/README.md [hunk 1]
```diff
@@ -70,8 +56,24 @@ B) IDE
 * Find the Rabbitmq container and click on the URL under Port(s) for the U.I. This should open up the RabbitMQ website on your default browser.
 * Login username and password should be "guest"
 * Alternative:  Click on one of the generated URLs in your terminal, such as "http://localhost:15672/", once your docker container is up and running.
-
-6. You are now ready to start the lab. Begin by navigating to the producer folder and reading the "README.md" file. Each of the three units will contain a readme file which will give you the necessary instructions to complete the lab and test your solution. The order in which the lab should be completed as well as information regarding rabbitmq can be found below.
```
Comment (RichardAF8) [inline Tech-Lab-On-Campus/README.md:74]: this should be after the jupyter notebook part

**Comment by RichardAF8** [comment]
the git clone part will change because now they are cloning their fork not the main repo..

File: Tech-Lab-On-Campus/README.md (hunk 1 above)
Comment (mariarivera30) [reply Tech-Lab-On-Campus/README.md:74]: Why this is not for both ? I think this part will be done after jupyter and after ide is set.

**Comment by mariarivera30** [comment]
> First step should be forking the repo with an image of clicking the fork button

missed this on my notes

File: Tech-Lab-On-Campus/README.md (hunk 1 above)
Comment (mariarivera30) [reply Tech-Lab-On-Campus/README.md:74]: I changed as requested.

File: Tech-Lab-On-Campus/README.md [hunk 2]
```diff
@@ -22,40 +22,29 @@ The idea of this lab is to offer exposure to the RabbitMQ messaging framework, p
...
+1. Fork the repo
+
+   ![fork](../data/fork.jpg)
+   
+2. Clone the forked repo into your working directory.
 ```sh
-git clone git@github.com:bloomberg/bbit-learning-labs.git
+git clone git@github.com:username/bbit-learning-labs.git
 ```
-
```
Comment (RichardAF8) [inline Tech-Lab-On-Campus/README.md:29]: I'd add 
* Note: please replace username with your github account username

File: Tech-Lab-On-Campus/README.md [hunk 3]
```diff
@@ -22,40 +22,29 @@ The idea of this lab is to offer exposure to the RabbitMQ messaging framework, p
...
 
-1. Clone the repo into your working directory.
+1. Fork the repo
+
+   ![fork](../data/fork.jpg)
+   
+2. Clone the forked repo into your working directory.
 ```sh
-git clone git@github.com:bloomberg/bbit-learning-labs.git
+git clone git@github.com:username/bbit-learning-labs.git
```
Comment (RichardAF8) [inline Tech-Lab-On-Campus/README.md]: git@github.com:[username]/bbit-learning-labs.git

Comment (RichardAF8) [reply Tech-Lab-On-Campus/README.md]: or maybe the step before this should be get your SSH key with a screenshot of where to get it circling the green code  button as well as the SSH key

Comment (RichardAF8) [reply Tech-Lab-On-Campus/README.md]: 3. Get your SSH key from your fork

File: Tech-Lab-On-Campus/README.md (hunk 2 above)
Comment (mariarivera30) [reply Tech-Lab-On-Campus/README.md:29]: I mean if I add the image to get the ssh then I dont think I need the note

**Pull request merged**